src/libs/cosmos.ts

```diff
@@ -461,6 +483,10 @@ window.cockpit = {
   registerActionCallback: registerActionCallback,
   unregisterActionCallback: unregisterActionCallback,
   executeActionCallback: executeActionCallback,
+  // Timers management:
+  setInterval: (callback: () => void, delay?: number) => cockpitTimerManager.createManagedSetInterval(callback, delay),
+  setTimeout: (callback: () => void, delay?: number) => cockpitTimerManager.createManagedSetTimeout(callback, delay),
```

Comment:
Would be good if we can match the interface for the real ones - returning ID values that allow user code to clear them if they want to (since that could be relevant to complex functionality).

Reply:
Agree.